app/views/shared/_header.html.slim

```diff
@@ -4,13 +4,15 @@
   i#open_main_menu.fa.fa-bars.fa-lg.pull-left.visible-xs
 
 .text-right.user-header
-  .search-header.hidden-xs
-    .header-search-form
-      = form_tag search_index_path, method: 'get', class: 'navbar-form' do
-        = search_field_tag 'search', params[:search], class: 'form-control search-field', placeholder: 'Search'
-        = hidden_field_tag 'type', 'repositories'
-        button[type="submit" class="btn btn-default"]
-          i.fa.fa-search
+  - if not current_page?(url_for controller: 'search')
```

Comment:
I added this conditional just to remove the search from the topbar on search page, because on the search page already have a search form (with the ajax filter)